### PR TITLE
feat!: Update to Testbench 8

### PR DESCRIPTION
**Pull request by alvarezguille**
with Selenium 4
update imports for changes classes
add osgi.annotation dependency in test module

### CODE REVIEW

File: flow-tests/test-root-context/pom.xml [hunk 1]
```diff
@@ -60,6 +60,13 @@
             <scope>provided</scope>
         </dependency>
 
+        <dependency>
+            <groupId>org.osgi</groupId>
+            <artifactId>osgi.annotation</artifactId>
```
Comment (Artur-) [inline flow-tests/test-root-context/pom.xml:65]: Why is this needed? Should all projects that use TestBench add this?

File: pom.xml [hunk 2]
```diff
@@ -102,7 +102,7 @@
         <maven.resources.plugin.version>3.0.2</maven.resources.plugin.version>
         <maven.clean.plugin.version>3.1.0</maven.clean.plugin.version>
         <maven.exec.plugin.version>1.6.0</maven.exec.plugin.version>
-        <testbench.version>7.0.3</testbench.version>
+        <testbench.version>8.0-SNAPSHOT</testbench.version>
```
Comment (Artur-) [inline pom.xml:105]: We should use an alpha here instead of a snapshot

Comment (alvarezguille) [reply pom.xml:105]: This is a draft PR validating that things work, once everything is green and the other changes are accepted I'll release an alpha, update the PR and mark it as ready for review.

File: flow-tests/test-root-context/pom.xml (hunk 1 above)
Comment (Artur-) [reply flow-tests/test-root-context/pom.xml:65]: Seems like it is related to the other use of OSGi in this module. If you remove `src/main/java/com/vaadin/flow/osgi` then you do not need this dependency either

**Review by Artur- — APPROVED** [review]
